DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/DependencyInjection/AutofacRhinoModule.cs

```diff
@@ -47,11 +51,35 @@ protected override void Load(ContainerBuilder builder)
     builder.RegisterType<RhinoSelectionBinding>().As<IBinding>().SingleInstance();
     builder.RegisterType<RhinoSendBinding>().As<IBinding>().SingleInstance();
 
+    // binding dependencies
+    builder.RegisterType<CancellationManager>().InstancePerDependency();
+    builder.RegisterType<SendBindingUICommands>().InstancePerDependency();
```

Comment:
I'm not sure we need to register this type directly. Maybe we should interface it out so it can be mocked 🤔 Binding commands and bindings are tightly coupled by definition anyway: if the SendBinding is not registered, using the `SendBindingUICommands` will do nothing at best.

Reply:
We would only need to interface out this type if we want to test the RhinoSendBinding class. In my opinion this isn't necessary (or at least it isn't urgent) because this class contains none of the Send logic. Also, it has a bunch of DUI3 and Rhino dependencies that would be tedious to abstract out like RhinoIdleManager and RhinoDocument events

Reply:
No longer needed after my last PR to this branch. Well get to inject these later on